src/store/thunks/users.js

```diff
@@ -15,6 +15,7 @@ export const signInWithGoogle = () => (dispatch) => {
           posts: [],
           full_name: res.user.displayName,
           avatar_link: "",
+          id: res.user.uid,
```

Comment:
Can we call this `userId` instead of `id`?

Reply:
definitely

Reply:
done